data/859/017/65/85901765.geojson

```diff
@@ -4,6 +4,8 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
+    "edtf:deprecated":"12-21-2020",
```

Comment:
Dudesti looks like a legitimate neighbourhood of Bucharest. Are we sure we want to deprecate this record?

Reply:
Dudesti is currently a street and it is used more as a street name than as a neighbourhood; areas adjacent to the street are also considered parts of nearby neighbourhoods such as Vitan and Dristor. 
I think that keeping it with the current geometry which spans massive areas in other neighbourhoods does more harm than good. 
Probably the best thing to do is to correct its centroid (which more correctly could be placed in this point: 44.421703, 26.131321 than in the one in the geoJson) and remove the polygon.